scene/resources/concave_polygon_shape_2d.cpp

```diff
@@ -32,6 +32,23 @@
 #include "servers/physics_2d_server.h"
 #include "servers/visual_server.h"
 
+bool ConcavePolygonShape2D::_edit_is_clicked(const Point2 &p_point, double p_tolerance) const {
+
```

Comment:
Is it checking for the inside of the polygon or only it's outline ?

Reply:
It only checks the outline as drawn in `ConcavePolygonShape2D::draw()`; it's consistent, as the inside is not drawn.